legal-api/src/legal_api/reports/business_document.py

```diff
@@ -120,8 +120,11 @@ def _get_template_data(self, get_json=False):
         try:
             # get document data
             business_json["reportType"] = self._document_key
-            business_json["business"] = self._legal_entity.json()
-            business_json["business"]["businessName"] = self._legal_entity.business_name
```

Comment:
I think we can revert back to this code.  `self._legal_entity.business_name` will need to be `self._business.business_name`.

Reply:
Ahhh yes. You are right

Reply:
actually, `business_name`  is broken.

You'll need to make this update, otherwise there are some scenarios where `business_name` breaks GPs.

![image](https://github.com/bcgov/lear/assets/6685223/33c938cb-c67e-4adf-aa98-9fff387ce9c8)